rundeck-authz/rundeck-authz-core/src/main/java/org/rundeck/core/auth/AuthResources.java

```diff
@@ -0,0 +1,232 @@
+package org.rundeck.core.auth;
+
+import java.util.*;
+
+public class AuthResources {
```

Comment:
It would be great if we could come up with a way to add type, kinds, actions etc... with a service loader or beans. That way plugins could add that info rather than having to hand edit this file every time we need something new. SCM and webhooks are prime candidates for that type of behavior. Don't know if we could work it into this refactor. What do you think?

Reply:
that would be a great enhancement.  i think for this phase i just want to deal with current usage